A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -142,12 +145,15 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         super.onUpdate(_deltaTime);
 
         const selfTestCurrentKnobValue = SimVar.GetSimVarValue(this.isTopScreen ? "LIGHT POTENTIOMETER:92" : "LIGHT POTENTIOMETER:93", "number");
+        // ECAM all button
+        const ecamAllButtonState = SimVar.GetSimVarValue("L:A32NX_ECAM_ALL_Push_IsDown", "Bool");
```

Comment:
I put this before the throttler as otherwise the state only updates when you hold the button long enough (~500ms)  to scroll through the pages with one click. Holding the button would work after the throttle as well